app/models/alchemy/element/definitions.rb

```diff
@@ -22,21 +22,10 @@ def definition_by_name(name)
 
       private
 
-      # Reads the element definitions file named +elements.yml+ from +config/alchemy/+ folder.
-      #
-      def read_definitions_file
-        if ::File.exist?(definitions_file_path)
-          ::YAML.load(ERB.new(File.read(definitions_file_path)).result) || []
-        else
-          raise LoadError, "Could not find elements.yml file! Please run `rails generate alchemy:scaffold`"
-        end
+      def loader
```

Comment:
I don't see a need for an extra loader method. Since we only use it once and it's already cached.